Release 2021.06.0 (#2184)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/apps/index.md

* The `grant_types` and `response_types` values described above are partially orthogonal, as they refer to arguments passed to different endpoints in the [OAuth 2.0 protocol](https://tools.ietf.org/html/rfc6749). However, they are related in that the `grant_types` available to a client influence the `response_types` that the client is allowed to use, and vice versa. For instance, a `grant_types` value that includes `authorization_code` implies a `response_types` value that includes `code`, as both values are defined as part of the OAuth 2.0 authorization code grant.

* A consent dialog box appears depending on the values of three elements:

  * `prompt` - a query parameter that is used in requests to [`/authorize`](/docs/reference/api/oidc/#authorize)
  * `consent_method` - An [application](/docs/reference/api/apps/#settings-7) property that allows you to determine whether a client is fully trusted (for example, a first-party application) or requires consent (for example, a third-party application).

>   * `consent` - a property on Scopes as listed in the table above. Allows you to individually enable and disable user consent for a scope.

Author: ```suggestion
  * `consent` - a Scope property, listed in the previous table, that allows you to enable or disable user consent for an individual scope.
```
